bundles/org.openhab.binding.mqtt.awtrixlight/README.md

```diff
@@ -0,0 +1,380 @@
+Switch Weather_Rainbow "Weather Rainbow Effect" (gAwtrixWeather) { channel="mqtt:awtrixapp:myBroker:myAwtrix:weather:rainbow" }
+
+// Custom App items with advanced features
+Switch Custom_Active "Custom App Active" (gAwtrix) { channel="mqtt:awtrixapp:myBroker:myAwtrix:custom:active" }
+String Custom_Text "Custom Text" (gAwtrix) { channel="mqtt:awtrixapp:myBroker:myAwtrix:custom:text" }
+String Custom_Icon "Custom Icon" (gAwtrix) { channel="mqtt:awtrixapp:myBroker:myAwtrix:custom:icon" }
+Color Custom_Color "Text Color" (gAwtrix) { channel="mqtt:awtrixapp:myBroker:myAwtrix:custom:color" }
+Color Custom_Background "Background Color" (gAwtrix) { channel="mqtt:awtrixapp:myBroker:myAwtrix:custom:background" }
+Number:Dimensionless Custom_ScrollSpeed "Scroll Speed" (gAwtrix) { channel="mqtt:awtrixapp:myBroker:myAwtrix:custom:scrollSpeed" }
+Switch Custom_Center "Center Text" (gAwtrix) { channel="mqtt:awtrixapp:myBroker:myAwtrix:custom:center" }
+Number:Dimensionless Custom_Progress "Progress [%d %%]" (gAwtrix) { channel="mqtt:awtrixapp:myBroker:myAwtrix:custom:progress" }
+Color Custom_ProgressColor "Progress Color" (gAwtrix) { channel="mqtt:awtrixapp:myBroker:myAwtrix:custom:progressColor" }
+```
+
+### Sitemap
+
+```
+
+sitemap awtrix label="Awtrix Display" {
+    Frame label="Display Control" {
+        Switch item=Display_Power
+        Slider item=Display_Brightness
+        Switch item=Display_Screen
+        Switch item=Display_Sound
+        Switch item=Display_AutoBrightness
+        Text item=Display_Temperature
+        Text item=Display_Humidity
+        Text item=Display_Battery visibility=[Display_LowBattery==ON]
+        Text item=Display_WiFi
+        Text item=Display_CurrentApp
+    }
+    
+    Frame label="Clock App" {
+        Switch item=Clock_Active
+        Text item=Clock_Text
+        Colorpicker item=Clock_Color
+        Slider item=Clock_Duration
+    }
+    
+    Frame label="Weather App" {
+        Switch item=Weather_Active
+        Text item=Weather_Text
+        Text item=Weather_Icon
+        Colorpicker item=Weather_Color
+        Switch item=Weather_Rainbow
+    }
+    
+    Frame label="Custom App" {
+        Switch item=Custom_Active
+        Text item=Custom_Text
+        Text item=Custom_Icon
+        Colorpicker item=Custom_Color
+        Colorpicker item=Custom_Background
+        Slider item=Custom_ScrollSpeed
+        Switch item=Custom_Center
+        Slider item=Custom_Progress
+        Colorpicker item=Custom_ProgressColor
+    }
+}
+
+```
+
+## Discovery
+
+The binding can automatically discover Awtrix devices that publish their status to the configured MQTT broker. Once a device is discovered, it will appear in the inbox. Default Awtrix apps can also be discovered if `discoverDefaultApps` is enabled on the bridge. This is however not recommended as the default apps cannot be controlled via this binding.
```

Comment:
This section should be above the bridge section.

Reply:
Done